fftpack.c

```diff
@@ -2309,7 +2303,7 @@ void cosqb(integer n, real *x, real *wsave)
     x[1] *= 4.f;
   } else if (n == 2) {
     x1 = (x[1] + x[2]) * 4.f;
-    x[2] = tsqrt2 * (x[1] - x[2]);
+    x[2] = 2 * M_SQRT2 * (x[1] - x[2]);
```

Comment:
Multiplying every time anew is premature pessimisation :) Even if most compilers will probably perform this multiplication at compile time, what did you think about a constant for this? You could aptly call it `M_SQRT8` for example.